Gemfile

```diff
@@ -19,14 +19,14 @@ gem 'elasticsearch-model', '~>0.1.0'
 gem 'elasticsearch-dsl', '~> 0.1.5'
 
 gem 'newrelic_rpm', '~> 4.2'
-gem 'airbrake', '~> 6.1'
+gem 'airbrake', '~> 7.1'
```

Comment:
@noremmie - heads up on this Airbrake upgrade. I don't see any potentially breaking changes, but you never know: https://github.com/airbrake/airbrake/blob/master/CHANGELOG.md

Reply:
Thanks, @MothOnMars. After the next i14y deploy perhaps we can tickle a known error to make sure it shows up in Airbrake.